pyxem/utils/signal.py

```diff
@@ -94,6 +94,13 @@ def transfer_navigation_axes(new_signal, old_signal):
     return new_signal
 
 
+def value2index(index, signal_axis):
```

Comment:
To stay consistent with hyperspy, probably but I changed this behavior a little bit in https://github.com/hyperspy/hyperspy/pull/3055 so that `value2index` just returns an integer if an int is passed to it.  I think it makes sense to have a single function in hyperspy.axes that just returns what the index would be if some value is passed using something like `isig`

Reply:
Ok, I see, there is such a function but it is private: https://github.com/hyperspy/hyperspy/blob/70ca0762a332a7239a4c1eb75001062ab6c7495d/hyperspy/axes.py#L407